tools/build_defs/scala/scala.bzl

```diff
@@ -26,6 +26,40 @@ def _adjust_resources_path(path):
     return dir_1 + dir_2, rel_path
   return "", path
 
+def _build_nosrc_jar(ctx, buildijar):
+  res_cmd = ""
+  for f in ctx.files.resources:
+    c_dir, res_path = _adjust_resources_path(f.path)
+    change_dir = "-C " + c_dir if c_dir else ""
+    res_cmd = "\n{jar} uf {out} " + change_dir + " " + res_path
+  ijar_cmd = ""
+  if buildijar:
```

Comment:
Yeah, I don't like it. But `scala_library` defines an ijar output always. I don't see how I can tell bazel that output is only generated when srcs are non-empty. So, what I do is just copy the (empty) jar to be the ijar. I do this with the flag so that we can accomodate `scala_macro_library` which never creates ijars.

So basically a desire for consistency with `java_library` and both `scala_library` and `scala_macro_library` motivated this choice.